tests/utils/test_cachito.py

```diff
@@ -298,3 +298,105 @@ def test_download_sources_bad_request_type(tmpdir):
+
+
+@responses.activate
+@pytest.mark.parametrize('cachito_request', (CACHITO_REQUEST_ID,
+                                             {'id': CACHITO_REQUEST_ID},)
+                         )
+def test_get_icm(cachito_request, caplog):
+    icm = dedent(
+        '''\
+        {
+          "metadata": {
+            "icm_version": 1,
+            "icm_spec": "https://link.to.icm.specification",
+            "image_layer_index": 3
+          },
+          "content_sets": [],
+          "image_contents": [
+            {
+              "purl": "pkg:golang/github.com%2Frelease-engineering%2Fretrodep%2Fv2@v2.0.2",
+              "dependencies": [
+                {
+                  "purl": "pkg:golang/github.com%2Fop%2Fgo-logging@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fmasterminds%2Fsemver@v1.4.2"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fpkg%2Ferrors@v0.8.1"
+                },
+                {
+                  "purl": "pkg:golang/gopkg.in%2Fyaml.v2@v2.2.2"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Ftools%2Fgo%2Fvcs@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Frelease-engineering%2Fretrodep%2Fv2%2Fretrodep"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Frelease-engineering%2Fretrodep%2Fv2"
+                }
+              ],
+              "sources": [
+                {
+                  "purl": "pkg:golang/github.com%2FMasterminds%2Fsemver@v1.4.2"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fkr%2Fpretty@v0.1.0"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fkr%2Fpty@v1.1.1"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fkr%2Ftext@v0.1.0"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fop%2Fgo-logging@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/github.com%2Fpkg%2Ferrors@v0.8.1"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Fcrypto@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Fnet@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Fsys@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Ftext@v0.3.0"
+                },
+                {
+                  "purl": "pkg:golang/golang.org%2Fx%2Ftools@v0.0.0"
+                },
+                {
+                  "purl": "pkg:golang/gopkg.in%2Fcheck.v1@v1.0.0"
+                },
+                {
+                  "purl": "pkg:golang/gopkg.in%2Fyaml.v2@v2.2.2"
+                }
```

Comment:
Is it possible to use fewer `purl` in this test data? Fewer lines would make it easier to read.

Reply:
Done